lib/TDL_provider.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:template/task.dart';
+import 'package:flutter/cupertino.dart';
+
+class TaskProvider extends ChangeNotifier {
+  final List<Tasks> _name = [];
+  String _filterBy = 'all';
+  List<Tasks> get getName {
+    return _name;
+  }
```

Comment:
Går att skriva inline.
```suggestion
  List<Tasks> get getName => _name;
```